locales/en/homepage.json

```diff
@@ -7,14 +7,14 @@
   "foss.button.label": "About the project",
   "foss.subtitle": "Open Terms Archive is free and open source software. Any entity can reuse and build upon it, as long as it shares its improvements with the community. The product is built collaboratively by all those who use and contribute to it.",
   "foss.title": "Contributive Digital Common",
-  "how.bloc1.desc": "The contractual documents of an online service are listed for tracking by volunteer contributors.",
-  "how.bloc1.title": "1. Identification",
-  "how.bloc2.desc": "Several times a day, our robots download and publicly archive the listed documents.",
-  "how.bloc2.title": "2. Archival",
-  "how.bloc3.desc": "When changes are spotted, they are recorded and exposed.",
-  "how.bloc3.title": "3. Detection",
-  "how.bloc4.desc": "All recorded versions are published in easy-to-analyse datasets.",
-  "how.bloc4.title": "4. Publishing",
+  "how.bloc1.desc": "The contractual documents of a service are targeted by volunteer contributors or partners.",
+  "how.bloc1.title": "1. Targeting",
+  "how.bloc2.desc": "Several times a day, our robots download and publicly archive the targeted documents.",
+  "how.bloc2.title": "2. Tracking",
+  "how.bloc3.desc": "When changes are spotted, they are recorded and exposed for human analysts.",
+  "how.bloc3.title": "3. Analysing",
+  "how.bloc4.desc": "All recorded versions are published in datasets enabling reuse and research.",
+  "how.bloc4.title": "4. Disseminating",
```

Comment:
Are you sure "Disseminating" is the best word for that? 
It's not really common for me. What about "Releasing" or "Spreading"?

Reply:
Good remark.
It's a bit specific indeed. Unfortunately, I did not find a better word: “spreading” is a bit ambiguous as a single verb (you could technically be “spreading” the data, whatever that means), and I was concerned that “releasing” questioned the publicity of the previous steps, as if the data was not released before it.
If you believe “Releasing” is clearer, still carries this idea of pushing for publicity and not merely making available, and does not contradict the “publicly archives” statement above, then let's change it to “Releasing”. Otherwise, please merge this PR :)

Reply:
Ok, let's keep "Disseminating"